networking/hardware_networks/configure-lacp-for-sriov.adoc

```diff
@@ -0,0 +1,16 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="sriov-lacp-sriov"]
+= Switch failure detection for bonded SR-IOV networks
```

Comment:
Is this heading too specific? Are there other use cases for this? 

If it is too specific, would something like "High-availability for bonded SR-IOV networks" work better?